Cli-Shared/Program.cs

```diff
@@ -964,14 +1065,15 @@ private static Credential ModalPromptForPassword(TargetUri targetUri, string mes
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
         private static void PrintVersion()
         {
-            Program.WriteLine($"{Title} version {Version.ToString(3)}");
+            Program.WriteLine($"{Title} version {Version}");
```

Comment:
Ah that was during my testing to be able to see #.#.#.# versions from the command line. I'll revert it.